tests/extensions/web/Microsoft.DotNet.UpgradeAssistant.Steps.Razor.Tests/MappedSubTextTests.cs

```diff
@@ -117,15 +117,15 @@ public async Task GetMappedSubTextsTests(string source, string? defaultMapPath,
 
                 new object?[]
                 {
-                    File.ReadAllText("MappedSubTextTestData.cs"),
+                    File.ReadAllText("MappedSubTextTestData.cs").ReplaceLineEndings(),
                     "Bar.cshtml",
```

Comment:
Hmm, perhaps we shouldn't even support tests on Mac for extensions that we disable on Mac anyway.

Reply:
The tests pass on macOS if we enable them with these changes.